File: C/excel_sheet_column_number.c
int titleToNumber(char * s){
    int num = 0;
        
    for(int i = 0; i < strlen(s); i++) {
        num = num + (s[i] - 'A' + 1) * pow(26, strlen(s)-i-1);
    }
        
    return num;
}

File: Python3/excel_sheet_column_number.py
class Solution:
    def titleToNumber(self, s: str) -> int:
        num = 0
        
        for i in range(len(s)):
            num = num + (ord(s[i]) - ord('A') + 1) * pow(26, len(s)-i-1) 
            
        return num
